src/System.Private.Xml/src/System/Xml/Serialization/Compilation.cs

```diff
@@ -307,7 +307,6 @@ internal static Assembly GenerateAssembly(XmlMapping[] xmlMappings, Type[] types
                 writer.WriteLine("#if _DYNAMIC_XMLSERIALIZER_COMPILATION");
```

Comment:
The AllowPartiallyTrustedCallers and SecurityTransparent attributes should be removed as well.